public/index.js

```diff
@@ -1,11 +1,7 @@
 var search_box = document.querySelector('#nav_search_box');
-search_box.addEventListener('input', search);
+search_box.addEventListener('change', search);
 function search(event){
-    searches = require('../exampleQuiz.json');
-    for(var i=0;i<searches.length;i++){
-        if((searches[i].name.includes(search_box.value)||(searches[i].description.includes(search_box.value))){
-            
-        }
-    }
-
+    var url='/search/'+search_box.value.replace(/\s+/g, '').toLowerCase();
```

Comment:
Can we undo this " "-replacement process? Do we even need to get spaces back on the server? 🤔

Reply:
Can't undo it but we could replace spaces with + or some other special character by replacing between (/\s+/g, '+'). I don't know why we would need spaces, unless you want to search the description for a whole sentence.

Reply:
Sounds good! I trust your judgment